#异常

Throwable是所有异常的父类

####语法
	try{..}
	catch{..}
	finally{..}
可以有零到多个 catch 块。finally 块可以省略。 但是 catch 和 finally 块至少应该存在一个

####异常表达式
try语句是一个表达式。表达式的值是try块中最后一个表达式的值，或者是catch块中最后一个表达式的值，finally块的内容不会影响表达式的结果

	val a: Int? = try { parseInt(input) } catch (e: NumberFormatException) { null }finally{}

####kotlin没有受检查异常
java中的异常分为**受检异常**（checked exception: 非RuntimeException的子类）和**非受检异常**（unchecked exception: RuntimeException的子类）。受检异常和非受检异常在功能上是等价的。
受检异常必须被捕获或者被传递给调用者。
